CodeEdit/NavigatorSidebar/SourceControlNavigator/SourceControlToolbar.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SourceControlToolbar.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+
+struct SourceControlToolbar<T: View>: View {
+
+    private let height: Double
+    private let content: () -> T
+
+    init(
+        height: Double = 27,
+        @ViewBuilder content: @escaping () -> T
+    ) {
+        self.height = height
+        self.content = content
+    }
+
+    var body: some View {
+        ZStack {
+            HStack {
+                content()
+                    .padding(.horizontal, 8)
+            }
+        }
+        .frame(height: height)
+    }
+}
```

Comment:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/806104/167056728-f969e203-fe86-4137-bbb3-fc8fbaa363c0.png">

If you notice from the designs and in Xcode, the selected segmented control items background is not the muted blue that you find elsewhere, in the sidebar it is a more prominent blue. I think it is a specific material. @lukepistrol I seem to remember a PR where you replaced our blue backgrounds with this material. We may need to add an flag in segmented control where if it is true, it uses the more prominent blue style. It may be better if we can even detect that it is in use in a navigator and do this automatically without a flag.

Reply:
@austincondiff @nanashili: regarding the background material: `EffectView.selectionBackground` is the one to use.